blazor/datagrid/columns.md

```diff
@@ -511,6 +511,69 @@ In the following code sample, you can prevent default filter query generation us
 
 > You can find the fully working sample [here](https://github.com/SyncfusionExamples/blazor-datagrid-prevent-query-generation-for-foriegnkey-column).
 
+### Customize filter UI in foreignkey column
+
+A custom filtering UI can be rendered using the [FilterTemplate](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Grids.GridColumn.html#Syncfusion_Blazor_Grids_GridColumn_FilterTemplate) feature of the Grid. The following example demonstrates the way to create a custom filtering UI in the foreign column.
```

Comment:
Need to add comment above the code example. 

"For all Filter type other than FilterBar, filtering parameters will be sent in the form of PredicateModel<T>.
 For foreign key column, T represents the type of ForeignKeyValue property. "